main_routers/music_router.py

```
@@ -65,6 +23,152 @@ def _patch_pyncm_async() -> None:
+        }
+
+        MAX_MUSIC_SIZE = 50 * 1024 * 1024  # 50MB 上限
+
+        async with httpx.AsyncClient(timeout=60.0, follow_redirects=True) as client:
+            resp = await client.get(decoded_url, headers=request_headers)
+            resp.raise_for_status()
+
+            content_type = resp.headers.get('Content-Type', 'audio/mpeg').split(';', 1)[0].strip()
+            if 'audio' not in content_type and 'video' not in content_type:
+                logger.warning(f"[Music Proxy] 非音频内容类型: {content_type}")
+                return JSONResponse(content={"success": False, "error": "音乐源返回了无效内容（非音频格式）"}, status_code=502)
```

Ruff (0.15.9)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)